### ISSUE TITLE
Add existing UI options to drush locale:import

### ISSUE THREAD

**Issue by davidferlay** [drush-ops/drush#4221]
**Current behavior**

- In Drupal 8 "Interface Translation" UI, tab "Import", user can select a po file, lang to import to, and most important : **3 boolean options**

 
![image](https://user-images.githubusercontent.com/20738264/66947776-c768d980-f053-11e9-9a1c-ade929138fad.png)

Using this UI to import po file manually works for all business cases I faced because the 3 boolean options make it flexible. It is however, a manual action.

- Let's look at Drush now : command `drush local:import` exists, but these 3 boolean options are missing.

![image](https://user-images.githubusercontent.com/20738264/66947857-ea938900-f053-11e9-90d6-c69324942f12.png)

**Describe the solution you'd like**

I believe it makes more sense to have these same import options in Drush rather than to use a contrib module.

Implementing the same 3 boolean options in the `drush local:import` command would allow to automate translation deployments in all scenarios i face, with only core Drush commands.

- **TLDR** : I propose to add the 3 missing import options existing in UI to Drush local:import command

**Describe alternatives you've considered**

I tested https://www.drupal.org/project/drush_language and although it partially answer my needs, it doesn't seem well maintained. In addition I would rather close the short gap in Drush than use one more overhead module.

**Additional context**

- This Drush command would be useful to any automated Drupal deployment scenario

**Comment by andypost** [comment]
So the real issue is strange - when you type `local:import` it not always can find this command
Guess autosuggester used some place

**Comment by Chi-teck** [comment]
It's also possible to execute that command by typing ` drush l:im` or drush `lo:i`.
That feature comes with Symfony Console, however apparently help command is not aware about command shortcuts. 
https://symfony.com/doc/current/components/console/usage.html#shortcut-syntax
